### PR TITLE
Bug 1212716 - Implement TV Browser IndexedDB adapter for Places DataStore. r=yifan

### PR DESCRIPTION
**Pull request by weilonge**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5756feec96bf93339a90277938ccedd3d28ac455

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=85ca2f62d735516a6b4252a5f8e0b291a4858ea1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=79477aa4a1df29a926b72dd110205fb94604e814

File: tv_apps/browser/manifest.webapp [hunk 1]
```diff
@@ -48,7 +48,7 @@
   },
   "datastores-access": {
     "places": {
-      "readonly": true,
+      "readonly": false,
```
Comment (weilonge) [inline tv_apps/browser/manifest.webapp]: for test purpose only

File: tv_apps/browser/js/datasync/sync_history.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+}
+*/
+
+var SyncHistory = (function () {
+  var syncDataStore;
+
+  function handleTask(task) {
+    return new Promise(resolve => {
+      var data = {};
+      console.log(task);
```
Comment (begeeben) [inline tv_apps/browser/js/datasync/sync_history.js]: Please remove the debug purpose console log.

File: tv_apps/browser/js/datasync/sync_db_helper.js [hunk 3]
```diff
@@ -213,6 +223,14 @@ var SyncBrowserDB = {
   },
 
   /**
+   * Get history by timestamp range.
```
Comment (begeeben) [inline tv_apps/browser/js/datasync/sync_db_helper.js]: Could we have the detail comment here as well? ``` * Get history by timestamp range at (start < record.timestamp <= end)```

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6af20c25640a80c0a60cf75f05a79cf33b7c69da

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=46003b95dcf64494e2e91abb86714f6c413e41a5

**Pull request merged**